### PR TITLE
Added improved pun module

### PR DESCRIPTION
**Pull request by J3RN**

### CODE REVIEW

**Comment by kaneda** [comment]
You're missing a copyright header, as well as the command priority

**Comment by J3RN** [comment]
@kaneda Thanks. Better now?

File: modules/pun.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+"""
+pun.py - jenni Pun Module
+Copyright 2009-2013, Michael Yanovich (yanovich.net)
```
Comment (kaneda) [inline modules/pun.py]: Move to 2015 dates and add yourself :P

**Comment by J3RN** [comment]
Done!

**Comment by kaneda** [comment]
@J3RN now squash it into a single commit and we're all set :)

**Comment by J3RN** [comment]
Alright, we should be good now. Git was being weird.

**Pull request merged**